arch/src/aarch64/mod.rs

```diff
@@ -20,6 +29,15 @@ pub struct EntryPoint {
     pub entry_addr: GuestAddress,
 }
 
+pub fn configure_vcpu(
+    #[allow(unused_variables)] fd: &VcpuFd,
+    #[allow(unused_variables)] id: u8,
+    #[allow(unused_variables)] kernel_entry_point: Option<EntryPoint>,
+    #[allow(unused_variables)] vm_memory: &GuestMemoryAtomic<GuestMemoryMmap>,
```

Comment:
Please use `_` in front of each variable name, instead of using `#[allow(unused_variables)]`